### PR TITLE
Feature/edit wisdom

### PR DESCRIPTION
**Pull request by BenRodProd**
view in iphone se mode

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -22,7 +22,10 @@ export default function App({ Component, pageProps }) {
   function handleNewWisdomSubmit(wisdom) {
     setCurrentLibrary([...currentLibrary, wisdom]);
   }
-
+  function handleEditWisdomSubmit(wisdom) {
+    const arrayToKeep = currentLibrary.filter((elem) => elem.id != wisdom.id);
```
Comment (ella-blaupause) [inline pages/_app.js:26]: == and != are non-strict equality operators. You should avoid them 99% of the time.
Non-strict equality tries to use type coercion to convert both values to the same type: "3" == 3 is true, which is seldomly what you want.

File: pages/library/[id].js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      id: wisdom.id,
+      book: currentBook,
+    });
+    router.push("/library/viewBook");
+  }
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
+  const router = useRouter();
+  const { id } = router.query;
```
Comment (ella-blaupause) [inline pages/library/[id].js:92]: A small recommendation:
By declaring const router = useRouter(); and const { id } = router.query; at the beginning of the component, you also improve code readability and avoid unnecessary repetitions of code when accessing the route object and dynamic route parameter at different parts of your component.

File: pages/library/viewBook.js [hunk 3]
```diff
@@ -107,11 +116,15 @@ export default function ViewBook({ library, currentBook }) {
...
+                          <p>{wisdom.question}</p>
 
-                      <h2>Answer:</h2>
-                      <p>{wisdom.answer}</p>
+                          <h2>Answer:</h2>
+                          <p>{wisdom.answer}</p>
+                        </div>
+                      </StyledLink>
 
                       <div className="seperator">
```
Comment (ella-blaupause) [inline pages/library/viewBook.js:129]: This is not part of the PR, but 🫣:
Why are you declaring a class here?
otherwise you used styled components and here you don't. you might want to use a consistent method for declaring components to keep the code readable and easy to maintain.

**Review by ella-blaupause — COMMENTED** [review]
Great work💪

❗️I recommend you to use !== instead of !=.
This helps to avoid unexpected results and can make the code more predictable and less error-prone.❗️

File: pages/library/viewBook.js (hunk 3 above)
Comment (BenRodProd) [reply pages/library/viewBook.js:129]: This was a recommandation for the previous pr, not my idea... for accessability reasons🤷

File: pages/_app.js (hunk 1 above)
Comment (BenRodProd) [reply pages/_app.js:26]: Oops, typo, thx

File: pages/library/viewBook.js (hunk 3 above)
Comment (ella-blaupause) [reply pages/library/viewBook.js:129]: Okay, then I did not say anything😅

Overall, fantastic job!

**Review by ella-blaupause — APPROVED** [review]
LGTM
